eve/main.yml

```diff
@@ -224,6 +224,13 @@ stages:
           command: bash cypress.sh
           workdir: build/ui
           haltOnFailure: true
+      - Upload:
+          name: Upload Cypress errors
+          source: build/ui
+          urls:
+            - 'cypress/screenshots/*'
+            - 'cypress/videos/*'
+          alwaysRun: true
```

Comment:
No, no need to run this step if the previous one failed

Reply:
Forget about it, in case of partial failure you'll still get some stuff pushed to artifacts. And it won't complain if there's nothing to upload.